app/views/categories/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="category-ctn">
+  <div class="category-nav">
+    <div class="nav-title-ctn">
+      <h3 class="nav-title">Categories</h3>
```

Comment:
- [ ] Kindly ensure your title is centered on the page, to follow the design pattern given, where every title is at the center:

<img width="204" alt="image" src="https://user-images.githubusercontent.com/61408860/189101040-86349bc0-227d-495f-9d25-d6375e702123.png">

Reply:
well resolved! ✔️